orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -15,164 +15,180 @@
-            </param>
-            <param>
-                <name>profiler-scheduler-api.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>ranger.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>rangerui.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>schema-registry.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>smm-ui.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>solr.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>sparkhistoryui.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>spark3historyui.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>webhdfs.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>yarnuiv2.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>resourcemanager.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>zeppelinui.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>zeppelinws.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-            <param>
-                <name>flink.acl</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
-            </param>
-        </provider>
-{% endif %}
-
+{%- if 'RANGER' in exposed and 'RANGER_ADMIN' in salt['pillar.get']('gateway:location') -%}
+    {% set ranger_installed = True %}
+    {% set aclsauthz_prefix = 'AclsAuthz.' %}
+{%- else -%}
+    {% set ranger_installed = False %}
+    {% set aclsauthz_prefix = '' %}
+{%- endif -%}
+
+{% if salt['pillar.get']('gateway:envAccessGroup') is defined and salt['pillar.get']('gateway:envAccessGroup') | length > 0 and salt['pillar.get']('gateway:enable_knox_ranger_authorizer') == True %}
```

Comment:
I think this `gateway:enable_knox_ranger_authorizer` check is at the wrong place because now it will be missing all the ACL checks on 7.1.0.  Probably it should be at line 37?

Reply:
Yes, you are right :(